neurons/validator.py

```diff
@@ -443,18 +445,38 @@ def update_models(self):
                 # Compare metadata and tracker, syncing new model from remote store to local if necessary.
                 updated = asyncio.run(self.model_updater.sync_model(hotkey))
 
-                if updated:
```

Comment:
This log should not have been behind this flag to be meaningful, but I found it to be quite noisy and removed it.

It is under trace though so can add it back in if you think it is useful. (And change "Updated model" -> "Synced model").